.github/workflows/ci.yaml

```diff
@@ -41,4 +43,4 @@ jobs:
           version: nightly
 
       - name: Run tests
-        run: FOUNDRY_PROFILE=ci forge test
\ No newline at end of file
+        run: make test-ci
```

Comment:
Added new make test-ci commands.

Reply:
I don't think we need a special command for this. Can we define the profile as an env var as shown [here](https://github.com/foundry-rs/forge-template/blob/d18f232927dba100b614230a88af59c67f62df0e/.github/workflows/ci.yml#L8-L9) and just have this run `make test`?

Reply:
Nice, updated it here https://github.com/ScopeLift/umbra-protocol/pull/341/commits/cf4f4c0e15648b55b029ee5d98e249781b3ee22d